src/cloud/lib/stores/electron.ts

```diff
@@ -20,6 +20,14 @@ import { IpcRendererEvent } from 'electron'
 import { useEffectOnce } from 'react-use'
 import ltSemver from 'semver/functions/lt'
 
+export function addToWebContents(eventName: string, callback: any) {
```

Comment:
Naming is bad. It should be addWebContentsEventListener.
Also, we should avoid using callback.

But also I don't like to have this method. Please check comments in boosthub-preload.js